cmd/satellite/open_stream.go

```diff
@@ -64,6 +64,7 @@ func NewOpenStreamCommand() *cobra.Command {
 				return err
 			}
 			if openStreamFlag.EnableAutoClose {
+				fmt.Print("Notice: The open stream flag '--enable-auto-close' which is supplied with an '--auto-close-time' has been deprecated and will be removed in a future release.\n\n")
```

Comment:
When we warn with no proxy + no outfile we use `log.Println`. Is there a reason for using `fmt.Print` here instead?

Reply:
I tried both options, and I thought it looks better to use fmt.Print which omits the date and time which is present at the start of every log message.